src/components/ViewerBorder/ViewerBorder.scss

```diff
@@ -0,0 +1,40 @@
+// TODO: all colors should be using variables and inheriting from sitebranding
+// TODO: all fonts should be inherited
+.viewer-border{
+  background-color: #f2f8fb;
+  display: inline-block;
+}
+.viewer-title{
+  min-width: 350px;
+  padding: 4px;
+  font-size: 14px;
+  font-weight: 600;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.71;
+  letter-spacing: 1px;
+  color: #292b2c;
+}
+.image-caption{
```

Comment:
It's called `image-caption` here, but `viewer-caption` in the render.

Reply:
ah thanks! good catch!